coffee_httpd/Cargo.toml

```
@@ -6,7 +6,9 @@ edition = "2021"
 [dependencies]
 actix-web = "4"
 clap = "4.1.11"
+tokio = { version = "1.22.0", features = ["full"] }
```

```suggestion
tokio = { version = "1.22.0", features = ["full"] }
```

we really need the full feature?